stdlib/collections/__init__.pyi

```diff
@@ -301,3 +310,10 @@ class ChainMap(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     def pop(self, key: _KT) -> _VT: ...
     @overload
     def pop(self, key: _KT, default: _VT | _T = ...) -> _VT | _T: ...
+    def copy(self: _S) -> _S: ...
+    @classmethod
+    @overload
+    def fromkeys(cls, iterable: Iterable[_T]) -> ChainMap[_T, None]: ...
+    @classmethod
+    @overload
+    def fromkeys(cls, iterable: Iterable[_T], *args: _S) -> ChainMap[_T, _S]: ...
```

Comment:
```
>>> ChainMap.fromkeys("abc", "x","y")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/akuli/.local/lib/python3.10/collections/__init__.py", line 1009, in fromkeys
    return cls(dict.fromkeys(iterable, *args))
TypeError: fromkeys expected at most 2 arguments, got 3
```

The args get passed to `dict.fromkeys()`, so its type should be copied from `builtins.pyi`, except that the first argument `iterable` is not positional-only in `ChainMap`.

Reply:
Yah, I was just looking at that. It [does have `*args` in the source code](https://github.com/python/cpython/blob/b0b10e146b1cbf9c5dfa44af116a2eeb0f210e8b/Lib/collections/__init__.py#L1015) for some strange reason, though, so let's hope stubtest doesn't complain about an inconsistency between the stub and the runtime.

Reply:
Both arguments _do_ have to be positional-only, because the second argument is a `*args` argument. So the better thing to do is to annotate the method as having two positional-only arguments, and add it to the stubtest allowlist so that it doesn't complain about a supposed inconsistency between the stub and the runtime.

Reply:
You can do `ChainMap.fromkeys(iterable="abc")`, but you're right, that doesn't apply to the second overload.

Reply:
Ohh, in that case I think it needs an additional overload.

Reply:
How's that look now?